File: Calculator.java
// Calculator.java
import java.util.Scanner;

public class Calculator{
	public static void main(String[] args){

		Scanner sc = new Scanner(System.in);

		double a = 10;
		double b = 5;
		char operator = '+';

		System.out.println("Zahl1 eingeben: ");
		a= sc.nextDouble();
		System.out.println("Zahl2 eingeben: ");
		b = sc.nextDouble();
		System.out.println("Operator eingeben: ");
		operator = sc.next().charAt(0);
		
		switch(operator){
			case '+':
				System.out.println("Resultat: "+(a+b));
				break;
			case '-':
				System.out.println("Resultat: "+(a-b));
				break;
			case '*':
				System.out.println("Resultat: "+(a*b));
				break;
			case '/':
				System.out.println("Resultat: "+(a/b));
				break;
			default:
				System.out.println("Error!");
				break;
		
		}

	}
}

File: Calculator.rb
# Calculator.rb

$result = 0

def add (a, b)
	$result = a + b
end

def sub (a, b)
	$result = a - b
end

def mul (a, b)
	$result = a * b
end

def div (a, b)
	$result = a / b
end

puts "Zahl1 eingeben: "
number1 = gets.to_i

puts "Zahl2 eingeben: "
number2 = gets.to_i

puts "Operator [+]|[-]|[*]|[/] eingeben"
operator = gets.chomp

case operator
	when "+" then add(number1, number2);
	when "-" then sub(number1, number2);
	when "*" then mul(number1, number2);
	when "/" then div(number1, number2);
end

print $result
print "\n"
